pkg/virt-api/webhooks/mutating-webhook/mutators/migration-create-mutator_test.go

```diff
@@ -102,11 +102,5 @@ func expectedMigrationObjectMeta(currentObjectMeta k8smetav1.ObjectMeta, vmiName
 }
 
 func expectedJSONPatchForVMIMCreation(expectedObjectMeta k8smetav1.ObjectMeta) ([]byte, error) {
-	return patch.GeneratePatchPayload(
-		patch.PatchOperation{
-			Op:    patch.PatchReplaceOp,
-			Path:  "/metadata",
-			Value: expectedObjectMeta,
-		},
-	)
+	return patch.New(patch.WithReplace("/metadata", expectedObjectMeta)).GeneratePayload()
```

Comment:
we can entirely get rid of this function, performing patch from the caller

Reply:
Done